app/models/work.rb

```diff
@@ -180,6 +180,22 @@ def new_recipients_allow_gifts
     end
   end
 
+  validate :parent_users_allow_related_works
+
+  def parent_users_allow_related_works
```

Comment:
Maybe `new_parents_can_be_cited`? I feel like `parent_users` might throw someone when seeing this validation without the context of the pull request title.